### PR TITLE
Check map extension

### PR DESCRIPTION
**Pull request by Zangetsu89**

### CODE REVIEW

File: includes/parser.h [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-/*   parser.h                                           :+:    :+:            */
-/*                                                     +:+                    */
-/*   By: abeznik <abeznik@student.codam.nl>           +#+                     */
-/*                                                   +#+                      */
-/*   Created: 2023/02/12 17:04:23 by abeznik       #+#    #+#                 */
-/*   Updated: 2023/02/15 14:55:57 by abeznik       ########   odam.nl         */
+/*                                                        :::      ::::::::   */
+/*   parser.h                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: edawood <edawood@student.42.fr>            +#+  +:+       +#+        */
```
Comment (ArthurBeznik) [inline includes/parser.h:6]: why are you stealing my files

File: includes/parser.h [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/02/12 17:04:23 by abeznik           #+#    #+#             */
+/*   Updated: 2023/02/15 17:22:24 by edawood          ###   ########.fr       */
```
Comment (ArthurBeznik) [inline includes/parser.h:9]: why are you stealing my files

File: includes/utils.h [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-/*   utils.h                                            :+:    :+:            */
-/*                                                     +:+                    */
-/*   By: abeznik <abeznik@student.codam.nl>           +#+                     */
-/*                                                   +#+                      */
-/*   Created: 2023/02/12 17:26:15 by abeznik       #+#    #+#                 */
-/*   Updated: 2023/02/14 14:04:09 by abeznik       ########   odam.nl         */
+/*                                                        :::      ::::::::   */
+/*   utils.h                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: edawood <edawood@student.42.fr>            +#+  +:+       +#+        */
```
Comment (ArthurBeznik) [inline includes/utils.h:6]: why are you stealing my files

File: srcs/parser/parser.c [hunk 4]
```diff
@@ -1,20 +1,30 @@
...
-/*   parser.c                                           :+:    :+:            */
-/*                                                     +:+                    */
-/*   By: abeznik <abeznik@student.codam.nl>           +#+                     */
-/*                                                   +#+                      */
-/*   Created: 2023/02/12 16:56:05 by abeznik       #+#    #+#                 */
-/*   Updated: 2023/02/15 14:55:36 by abeznik       ########   odam.nl         */
+/*                                                        :::      ::::::::   */
+/*   parser.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: edawood <edawood@student.42.fr>            +#+  +:+       +#+        */
```
Comment (ArthurBeznik) [inline srcs/parser/parser.c:6]: why are you stealing my files

File: srcs/utils/error_msg.c [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-/*   error_msg.c                                        :+:    :+:            */
-/*                                                     +:+                    */
-/*   By: abeznik <abeznik@student.codam.nl>           +#+                     */
-/*                                                   +#+                      */
-/*   Created: 2023/02/14 13:52:51 by abeznik       #+#    #+#                 */
-/*   Updated: 2023/02/14 14:06:26 by abeznik       ########   odam.nl         */
+/*                                                        :::      ::::::::   */
+/*   error_msg.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: edawood <edawood@student.42.fr>            +#+  +:+       +#+        */
```
Comment (ArthurBeznik) [inline srcs/utils/error_msg.c:6]: why are you stealing my files

File: srcs/utils/logger.c [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
-/*   logger.c                                           :+:    :+:            */
-/*                                                     +:+                    */
-/*   By: abeznik <abeznik@student.codam.nl>           +#+                     */
-/*                                                   +#+                      */
-/*   Created: 2023/02/14 13:54:14 by abeznik       #+#    #+#                 */
-/*   Updated: 2023/02/15 14:55:46 by abeznik       ########   odam.nl         */
+/*                                                        :::      ::::::::   */
+/*   logger.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: edawood <edawood@student.42.fr>            +#+  +:+       +#+        */
```
Comment (ArthurBeznik) [inline srcs/utils/logger.c:6]: WHY???????????

File: srcs/parser/map_name_extention_checker.c [hunk 7]
```diff
@@ -0,0 +1,35 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   extention.c                                        :+:      :+:    :+:   */
```
Comment (ArthurBeznik) [inline srcs/parser/map_name_extention_checker.c:4]: extension?

Comment (ArthurBeznik) [reply srcs/parser/map_name_extention_checker.c:4]: quite a long name for the file dont you think? check_infile_extension ? check_file_extension ?

**Review by ArthurBeznik — CHANGES_REQUESTED** [review]
extenSion not extenTion, and the infile is being called the scene description file in the subject, if that can be of any use

File: srcs/parser/map_name_extention_checker.c (hunk 7 above)
Comment (Zangetsu89) [reply srcs/parser/map_name_extention_checker.c:4]: Can't agree more xD

**Review by ArthurBeznik — COMMENTED** [review]
great stuff

**Pull request merged**